### DISCUSSION TITLE
Systemd support is now available in WSL!

### DISCUSSION THREAD

**Discussion by TaivasJumala** [arkane-systems/genie#300] [General]
[Systemd support is now available in WSL!](https://devblogs.microsoft.com/commandline/systemd-support-is-now-available-in-wsl/)

**Comment by cerebrate** [comment]
Yep, I know.

Already working on the necessary hacks for _that_ : see https://github.com/arkane-systems/bottle-imp .

(Covering the other things genie does that aren't just running systemd in a container; see also issues Microsoft/WSL#8840, Microsoft/WSL#8842, Microsoft/WSL#8844, Microsoft/WSL#8846, etc.)

**Comment by cerebrate** [comment]
...and bottle-imp has now shipped!

**Reply by TaivasJumala** [reply]
Sick.
> Keeps the WSL instance running even when you have no active sessions.

I like this.
